code/API_definitions/predictive-connectivity-data-vwip.yaml

```
@@ -0,0 +1,1045 @@
+            (i.e. which allows 2023-07-03T14:27:08.312+02:00 or 2023-07-03T12:27:08.312Z)
+          example: "2023-07-03T10:00:00Z"
+        endTime:
+          type: string
+          format: date-time
+          description: >-
+            Interval end time. It must follow [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6)
+            and must have time zone. Recommended format is yyyy-MM-dd'T'HH:mm:ss.SSSZ
+            (i.e. which allows 2023-07-03T14:27:08.312+02:00 or 2023-07-03T12:27:08.312Z)
+          example: "2023-07-03T11:00:00Z"
+        cellConnectivityData:
+          $ref: '#/components/schemas/CellConnectivityDataArray'
+      required:
+        - startTime
+        - endTime
+        - cellConnectivityData
+    CellConnectivityDataArray:
+      type: array
+      description: >-
+        Connectivity data for the different cells in a concrete time range.
+      items:
+        $ref: '#/components/schemas/CellConnectivityData'
+      minItems: 1
+    CellConnectivityData:
+      description: >-
+        Represents a rectangular grid cell within the specified area, including
+        its connectivity data.
+      properties:
+        geohash:
+          type: string
+          description: >-
+            Coordinates of the cell represented as a string using the [Geohash
+            system](https://en.wikipedia.org/wiki/Geohash), encoding a
+            geographic location into a short string. The value length, and thus
+            the cell granularity, is determined by the implementation.
+          example: ezdmemd
+        layerConnectivities:
+          $ref: '#/components/schemas/LayerConnectivities'
+      required:
+        - geohash
+        - layerConnectivities
+    LayerConnectivities:
+      type: array
+      description: >-
+        Sequence of connectivity quality values in each of the layers in which
+        the vertical space over the area determined by the `geohash` is
+        decomposed. All layers are considered having the same `layerThickness`
+        meters and are ordered consecutively in the array starting from the
+        ground (First layer will start at 0 meters AGL). So the layer at
+        index = N would comprise the heights, referenced to AGL, among N *
+        `layerThickness` meters and (N+1) * `layerThickness` meters.
+
+
+        The possible connectivity values are:
+          - GC: Means good connectivity. Service level met even in unfavorable conditions.
+          - MC: Means marginal connectivity. Service level might not be met.
+          - NC: Means no connectivity. Service level not met.
+          - ND: No connectivity data.
+
+        **NOTE**: These connectivity values are relative to the `serviceLevel`
+        and `networkType` specified in the request.
+      items:
+        $ref: '#/components/schemas/Connectivity'
+    Connectivity:
+      type: string
+      enum:
+        - GC
+        - MC
+        - NC
+        - ND

```

Add "null" value indicating that only a signal strength estimate is being provided, and not a connectivity quality estimate
```suggestion
        - ND
        - SS
```